docs/src/apimanual.md

```diff
@@ -566,7 +589,7 @@ new coefficients are known.
 
 ### Problem data
 
-All data passed to the solver should be copied immediately to internal data structures. Solvers may not modify any input vectors and should not assume that input vectors will not be modified by users in the future. This applies, for example, to the coefficient vector in `ScalarAffineFunction`. Vectors returned to the user, e.g., via `ObjectiveFunction` or `ConstraintFunction` attributes should not be modified by the solver afterwards. The in-place version of `get!` can be used by users to avoid extra copies in this case.
+All data passed to the solver should be copied immediately to internal data structures. Solvers may not modify any input vectors and should not assume that input vectors will not be modified by users in the future. This applies, for example, to the terms vector in `ScalarAffineFunction`. Vectors returned to the user, e.g., via `ObjectiveFunction` or `ConstraintFunction` attributes should not be modified by the solver afterwards. The in-place version of `get!` can be used by users to avoid extra copies in this case.
```

Comment:
To avoid the double negative: "and should assume that input vectors may be modified by users in the future"

s/the terms vector/the `terms` vector

I kind of think there should be a comma after attributes, but then there is a lot of commas.
s/attributes should not be/attributes, should not be